apis/supernova/src/handler/data-property-handler.js

```diff
@@ -664,90 +678,94 @@ class DataPropertyHandler {
   /**
    * Adds a library measure to the handler.
    * @private
-   * @param {LibraryMeasure} libraryMeasure
+   * @property {string} id
+   * @property {qix.NxMeasure=} defaults
    * @returns {Promise<qix.NxMeasure=>} The result of addMeasure.
    * @description Creates and adds a library measure.
    * @memberof DataPropertyHandler
    * @example
-   * handler.addLibraryMeasure({ id: 'measId', defaults: { qDef: { ... } } });
+   * handler.addLibraryMeasure('measId', { qDef: { qNumFormat: { qType: 'F' } } });
    */
-  addLibraryMeasure(libraryMeasure) {
-    const measure = this.createLibraryMeasure(libraryMeasure);
+  addLibraryMeasure(id, defaults) {
+    const measure = this.createLibraryMeasure(id, defaults);
     return this.addMeasure(measure);
   }
 
   /**
    * Adds multiple library measures to the handler.
    * @private
-   * @param {LibraryMeasure[]} libraryMeasures - Array of library measure.
+   * @param {LibraryMeasure[]} args - Array of library measures.
    * @returns {Promise<qix.NxMeasure[]>} The result of addMeasures.
    * @description Creates and adds multiple library measures.
    * @memberof DataPropertyHandler
    * @example
-   * handler.addLibraryMeasures([{ id: 'A' }, { id: 'B', defaults: { qDef: { ... } } }]);
+   * handler.addLibraryMeasures([{ id: 'A' }, { id: 'B', defaults: { qDef: { qNumFormat: { ... } } } }]);
    */
-  addLibraryMeasures(libraryMeasures) {
-    const measures = libraryMeasures.map((libraryMeasure) => this.createLibraryMeasure(libraryMeasure));
-    return this.addMeasures(measures);
+  addLibraryMeasures(args) {
+    const measures = args.map(({ id, defaults }) => this.createLibraryMeasure(id, defaults));
+    return this.addMeasures(measures, false);
```

Comment:
Why make the switch to "args" as the name of the array?
args is generally used for unknown arguments that needs a check, but this needs to always be an array right?